src/PortfolioPlanning/Components/Plan/DependencyPanel.tsx

```diff
@@ -156,7 +163,11 @@ export class DependencyPanel extends React.Component<IDependencyPanelProps, IDep
                     total:
                         this.props.progressTrackingCriteria === ProgressTrackingCriteria.CompletedCount
                             ? dependency.TotalCount
-                            : dependency.TotalEffort
+                            : dependency.TotalEffort,
+                    showInfoIcon: this._showInfoIcon(dependency, dependsOn),
+                    infoMessage: dependsOn
+                        ? "Target date is later than " + this.props.workItem.title + "'s target date"

```

Comment:
If the panel is open for X, then work item Y in the dependsOn group should have target dates that are before the start date of X. So, "Target date is later than X's start date".